src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/ManagedResourceParserTests.cs

```diff
@@ -460,6 +460,7 @@ public void UpdateLayoutIdIsIncludedInDesigner ([Values(true, false)] bool useRt
 		}
 
 		[Test]
+		[Category ("SmokeTests")]
```

Comment:
This test doesn't actually build a full project, it only runs `<Aapt*/>` MSBuild tasks directly. It took 501ms for me locally.

Should we pick a different, _slower_ test, here?

Reply:
I added a couple of other more comprehensive tests. I think we can keep this around to make sure we have some coverage for our individual Task tests in this category?